apps/frontend/components/FormValidation/FloodFormValidation/FloodTables.tsx

```diff
@@ -28,6 +71,76 @@ export const FloodTables = ({
         onChange={onChange}
         isEditable={isEditMode}
       />
+      <DisasterTable
```

Comment:
Could this be a mapping instead? From a list of [{columnsm columnGroup}], or these could even be grouped before their export?

Reply:
to leverage a map?

Reply:
They are some specificity for dropdowns (passing the intl hooks) and multiple choices and this mapping will be different for the reports t so I don't know if it will be way more effective